company_website/tests/test_views.py

```diff
@@ -25,12 +29,38 @@ def test_that_get_context_data_return_bosses_and_employees_which_are_working(sel
 
 
 class EstimateProjectViewTests(TestCase):
-    def test_that_estimate_project_view_should_redirect_to_thank_you_view(self):
-        form_data = {
+    def setUp(self) -> None:
+        self.form_data = {
             "name": "John Doe",
             "email": "email@address.com",
             "idea_description": "Some description",
             "privacy_policy_accepted": True,
         }
-        response = self.client.post(path=reverse("estimate_project"), data=form_data, follow=True)
+        self.expected_mocked_mail_called = {
+            "email_data": {
+                "name": "John Doe",
+                "email": "email@address.com",
+                "country": "",
+                "idea_description": "Some description",
+                "nda_required": "",
+                "privacy_policy_accepted": True,
+                "project_development": "",
+                "monthly_bugdet": "",
+                "project_duration": "",
+                "design_product": "",
+                "company_info_origin": "",
+            },
+            "email_subject": ESTIMATE_PROJECT_EMAIL_SUBJECT,
+            "template": ESTIMATE_PROJECT_EMAIL_TEMPLATE_NAME,
+        }
+
+    def test_that_estimate_project_view_should_redirect_to_thank_you_view(self):
+        response = self.client.post(path=reverse("estimate_project"), data=self.form_data, follow=True)
```

Comment:
It should also has mock on send_mail method

Reply:
Done